### DISCUSSION TITLE
Problem in sending topic to change LWT temp when cooling

### DISCUSSION THREAD

**Discussion by westminzer** [Arnold-n/P1P2MQTT#62] [Q&A]
Hi all,

first of all a special greeting to Arnold. Some months ago I bought the P1P2 serial and now I'm finally able to control my Daikin Altherma 2 remotely through home assistant.

Everything works fine but I have a problem when I send a topic to change LWT set when in cooling mode. My pump is set to heat/cool with leaving water temp target (no room temp control). When I set it to "Heating" and, for example, I send "E 36 6 12C" target temp correctly changes to 30°C and all the ranges between 25°C to 60°C work. Otherwise, if it is set to cooling, and I send "E 36 6 A0" it should change to 16°C but actually nothing happens, I tried all the ranges from 8°C to 22°C with no result! Am I doing something wrong?

Hoping someone can help me.

Thank you :)

**Comment by Arnold-n** [comment]
Hi @westminzer, thanks and greetings! I do not use cooling myself but I suspect that it may require the use of another parameter than "36 6"; perhaps it is just "36 7", but I would not recommend to just try writing to random parameter numbers. Instead, could you change the LWT temperature on the main Daikin controller while logging P1P2/R/#, and share the log or check the log for changes in the 00F036* messages? I would expect that you see one parameter change  (or sometimes 2, or 4 parameters at the same time)  in response to your manual action, and that parameter (or one of these) can then hopefully be used for setting LWT in cooling mode.

**Reply by westminzer** [reply]
Hi @Arnold-n, I did as you suggest, I read the log and you were right: the correct parameter to set cooling temp is 7. I did some tries and it works. 
If you want to update your documentation I give you the correct model of my pump: EHVX08S26CB9W. Till now I can completely:
- Turn on/off DHW and LWT
- Activate DHW Booster
- Set temp of DHW, LWT in heating and now in cooling.

Thank you again, now I can control cooling settings too... really Happy :)

**Reply by Arnold-n** [reply]
Hi @westminzer, many thanks for trying and sharing! I added the new info to the Eseries code v0.9.33a, and renamed a few for consistency/self-explanation/shortness. Also removed some of the duplicate values from reporting to HA, so some old names may stick (retained in MQTT communication)  unless manually deleted from the MQTT server.

**Reply by stomko11** [reply]
@westminzer would you mind sharing what you have? I am struggling to understand how to operate my HP, started a discussion here: https://github.com/Arnold-n/P1P2Serial/discussions/96

**Comment by Arnold-n** [comment]
HA control is now available, so closing discussion here.
